pkg/costmodel/cluster.go

```diff
@@ -518,10 +541,13 @@ func ClusterNodes(cp cloud.Provider, client prometheus.Client, duration, offset
 	ramSystemPctMap := buildRAMSystemPctMap(resNodeRAMSystemPct)
 
 	cpuBreakdownMap := buildCPUBreakdownMap(resNodeCPUModeTotal)
-	activeDataMap := buildActiveDataMap(resActiveMins, resolution, cp.ParseID)
 	preemptibleMap := buildPreemptibleMap(resIsSpot, cp.ParseID)
 	labelsMap := buildLabelsMap(resLabels)
 
+	costTimesMinuteAndCount(activeDataMap, cpuCostMap, cpuCoresMap)
+	costTimesMinuteAndCount(activeDataMap, ramCostMap, ramBytesMap)
+	costTimesMinute(activeDataMap, gpuCostMap) // there's no need to do a weird "nodeIdentifierNoProviderID" type match since gpuCounts have a providerID
```

Comment:
So, each individual GPU gets counted separately? Interesting, I don't think I knew that.

Reply:
The GPU counts get multiplied in at line 531

Reply:
Ah, ok, the PR has changed since this comment. Will re-review.